api_tests/entities/update_labels.test.coffee

```diff
@@ -32,3 +32,18 @@ describe 'entities:update-labels', ->
     .catch undesiredErr(done)
 
     return
+
+  it 'should reject an up-to-date value', (done)->
+    humanPromise
+    .then (human)->
+      updateLabel human._id, 'en', 'foo'
+      .catch undesiredErr(done)
+      .then -> updateLabel human._id, 'en', 'foo'
```

Comment:
suggestion : 
```
    humanPromise
    .then (human)->
      humanLabel = human.labels.en
      updateLabel human._id, 'en', humanLabel
```

Reply:
`'foo'` represents a new value, not the old one: the old one isn't needed for labels update (at the difference of claims) as there is only one value per language: it could benefit from being renamed `set-label` instead of `update-label`